src/components/EventDetails.js

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
 import { Button } from './Button';
 import { SimpleMap } from './SimpleMap';
+import { Spinner } from './Spinner';
 
 export const EventDetails = (props) => {
-  return (
-    <div className='event-details-card'>
-      <div className="event-details-container">
-        <h3>{props.event.name.text}</h3>
-        {
-          props.event.logo.url ? 
-          <img alt="Event" src={props.event.logo.url} />
-          : null
-        }
+  return(
+    props.eventError ?  <div>Sorry, there seems to be an error. Please try again later.</div>
```

Comment:
(Optional) We could split this ternary logic out into separate functions, so that the component strictly handles the rendering and not the decision about what to render. We haven't stuck to this principle throughout the code though, so for speed's sake we could leave this as is.